client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -21,19 +22,22 @@ class ConversationCaterpillarComponent extends React.Component {
 
 	render() {
 		const { comments, translate } = this.props;
-		const lastComment = last( comments );
-		const lastCommenterName = get( lastComment, 'author.name' );
-		const commentCount = comments.length;
+		const commentCount = size( comments );
```

Comment:
@fraying explained in Slack:

> I was assuming the number was all comments for that post, minus however many are expanded already.
So, a post with 100 comments total, with three expanded in Convos, would be 97
A post with 100 comments total, with 10 expanded on the Fullpost page, the number would be 90
In Convos, if you closed all the comments, that 100 comment post would show 100.

Sounds like we'll have to pass the count into the component eventually from a parent component. I'll leave this as it is for the moment.